Tests/js/models/assets/ez-usermodel-tests.js

```diff
@@ -40,8 +89,108 @@ YUI.add('ez-usermodel-tests', function (Y) {
         tearDown: function () {
             this.model.destroy();
             delete this.model;
-        }
+        },
+
+        "Sync 'read' should load the content with CAPI": function () {
+            var m = this.model,
+                modelId = "/api/v2/ezp/model/mid",
+                callback = function () { };
+
+            Y.Mock.expect(this.capiMock, {
+                method: this.capiGetService,
+                returns: this.serviceMock
+            });
+            Y.Mock.expect(this.serviceMock, {
+                method: this.serviceLoad,
+                args: [
+                    modelId,
+                    Y.Mock.Value.Function
+                ],
+                run: Y.bind(function (id, callback) {
+                    callback(false, this.loadResponse);
+                }, this)
+            });
+
+            m.set('id', modelId);
+            m.sync('read', {
+                    api: this.capiMock
+                }, callback
+            );
+
+            Y.Mock.verify(this.capiMock);
+            Y.Mock.verify(this.serviceMock);
+        },
+
+        "Should not set avatar attribute with CAPI when user doesn't have avatar": function () {
+            var m = this.model,
+                modelId = "/api/v2/ezp/model/mid",
+                callback = function () { };
+
+            this.imageIdentifiers = [];
```

Comment:
I guess you did not you check the coverage. This is not enough because the `contentType` mock is still configured with the value you set in setUp.